### DISCUSSION TITLE
Creating and styling an animated logo for a Quarto site with CSS – Ella Kaye

### DISCUSSION THREAD

**Discussion by giscus** [EllaKaye/ellakaye-quarto#12] [General]
# Creating and styling an animated logo for a Quarto site with CSS – Ella Kaye

How and why I created an animated logo for this site, using CSS

https://ellakaye.co.uk/posts/2023-04-11_animated-logo/

**Comment by edenian-prince** [comment]
This is really awesome and helpful - thank you so much!!

**Reply by EllaKaye** [reply]
I’m so glad you found this helpful! Thank you so much for letting me know - I really appreciate it.
